### PR TITLE
Show number of competitors

### LINKED ISSUE #2248: Total actual competitor count on General Info page
After a competition's results have been posted, please include total competitor count on the General Info page.

This number is useful for planning purposes.

Right now there's no easy way to get this number. You can look up 3x3 Round 1 count, but not EVERYONE always competes in that, so this number is not necessarily correct.  Trying to count unique competitors manually across different events is onerous.

### PR DESCRIPTION
**Pull request by pedrosino**
Fixes #2248 

I asked @jonatanklosko to try it out since I can't do it now.

### CODE REVIEW

**Comment by pedrosino** [comment]
image

File: WcaOnRails/app/views/competitions/_competition_info.html.erb [hunk 1]
```diff
@@ -75,6 +75,12 @@
       </dd>
     </dl>
 
+	<dl class="dl-horizontal">
```
Comment (jfly) [inline WcaOnRails/app/views/competitions/_competition_info.html.erb]: The indentation looks screwy here?

File: WcaOnRails/app/views/competitions/_competition_info.html.erb [hunk 2]
```diff
@@ -75,6 +75,12 @@
...
+	  <dt><%= t 'competitions.nav.menu.competitors' %></dt>
+	  <dd><%= @competition.competitors.count %></dd>
+	  </dd>
```
Comment (jfly) [inline WcaOnRails/app/views/competitions/_competition_info.html.erb]: What is this `</dd>` closing? Is it an orphan?

File: WcaOnRails/app/views/competitions/_competition_info.html.erb (hunk 1 above)
Comment (pedrosino) [reply WcaOnRails/app/views/competitions/_competition_info.html.erb]: Sorry, I was using a not-that-great editor.

File: WcaOnRails/app/views/competitions/_competition_info.html.erb (hunk 2 above)
Comment (pedrosino) [reply WcaOnRails/app/views/competitions/_competition_info.html.erb]: I think I had the <dd> split into lines, but then moved it to a single one and forgot to remove this.

**Pull request merged**